File: C#/백준/Bronze/23804. 골뱅이 찍기 － ㄷ/골뱅이 찍기 － ㄷ.cs
using System;
using System.Text;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            StringBuilder sb = new StringBuilder();
            for (int i=0;i<5;i++){
                for (int j=0;j<a;j++){
                    if (i==0 || i==4){
                        for (int k=0;k<a*5;k++){
                            sb.Append("@");
                        }
                    }
                    else{
                        for (int k=0;k<a;k++){
                            sb.Append("@");
                        }
                    }
                    sb.AppendLine();
                }
            }
            Console.WriteLine(sb.ToString());
        }
    }
}

File: Java/백준/Bronze/23804. 골뱅이 찍기 － ㄷ/골뱅이 찍기 － ㄷ.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        StringBuilder sb = new StringBuilder();
        for (int i=0;i<5;i++){
            for (int j=0;j<a;j++){
                int n = 0;
                if (i==0 || i==4){
                    n = a*5;
                }
                else{
                    n = a;
                }
                for (int k=0;k<n;k++){
                    sb.append("@");
                }
                sb.append("\n");
            }
        }
        System.out.println(sb.toString());
    }
}